core_v2/sources/domain_e2e_tests.move

```diff
@@ -456,53 +456,6 @@ module aptos_names_v2::domain_e2e_tests {
         assert!(option::is_none(&domains::get_reverse_lookup(user_addr)), 1);
     }
 
-    #[test(
-        router_signer = @router_signer,
-        aptos_names_v2 = @aptos_names_v2,
-        user = @0x077,
-        aptos = @0x1,
-        rando = @0x266f,
-        foundation = @0xf01d
-    )]
-    fun test_admin_force_seize_domain_name_doesnt_clear_unrelated_primary_name(
```

Comment:
is it possible to keep this test and fix it?

Reply:
I was thinking about it, but I think this test is a bit weird and since we remove the primary name logic in the domains.move, we shouldn't have the test here. I added a test to the router part.